src/Microsoft.ApplicationInsights/DataContracts/ExceptionTelemetry.cs

```diff
@@ -286,32 +337,44 @@ private void ConvertExceptionTree(Exception exception, ExceptionDetails parentEx
             }
         }
 
-        private void UpdateExceptions(Exception exception)
+        private void UpdateData(Exception exception, ExceptionInfo exceptionInfo)
```

Comment:
>UpdateData [](start = 21, length = 10)

only one of the argument is valid for this method. Consider creating 2 separate methods `UpdateDataWithException()` and `UpdateDataWithExceptionInfo()` for this.

Reply:
Good catch, thank you. It fell apart during refactoring and I never noticed :)